src/EA.Weee.Web/Areas/Scheme/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -118,7 +121,7 @@ public ActionResult Transfer(Guid organisationId, int complianceYear)
                         NoteStatus.Rejected,
                         NoteStatus.Void,
                         NoteStatus.Returned
-                    }, SelectedComplianceYear(currentDate, manageEvidenceNoteViewModel), new List<NoteType>() { NoteType.Evidence, NoteType.Transfer }, false));
```

Comment:
Add unit tests for mapper being called with the configuration setting and page. Think the internal AA manage evidence notes have similar.